settings.py

```diff
@@ -5,6 +5,7 @@
 
 REPOSITORY = os.getenv("INPUT_REPOSITORY")
 GH_TOKEN = os.getenv("INPUT_GH_TOKEN")
+GH_API_URL = os.getenv("GITHUB_API_URL")
```

Comment:
Update the README, adding this new env var in the docs

Reply:
Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

Reply:
Same problem as before, if you don't put it in the docs, you've implemented a cool feature that no one will know